src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -24,20 +24,21 @@ public partial class Process : IDisposable
         public static Process[] GetProcessesByName(string? processName, string machineName)
         {
             ProcessManager.ThrowIfRemoteMachine(machineName);
-            if (processName == null)
-            {
-                processName = string.Empty;
-            }
 
-            var processes = new List<Process>();
+            processName ??= "";
```

Comment:
Just curious: is there any reason for now using "" instead of string.Empty?

Reply:
Just consistency, e.g. a test using InlineData to feed inputs can't use string.Empty, and since I was adding additional uses, I made them all "".

Reply:
@Nuklon i don't believe there is ever a reason to use String.Empty. It is an anachronism.

Reply:
Thanks!